fplus-lib/src/core/mod.rs

```diff
@@ -3426,22 +3426,30 @@ _(OLD vs NEW)_
         let mut id = String::new();
         let mut signing_address = String::new();
         let mut message_cid = String::new();
+        let mut increase_allowance_cid: Option<String> = None;
 
         if let Some(allocation) = active_allocation {
             datacap_allocation_requested.clone_from(&allocation.amount);
             id.clone_from(&allocation.id);
 
             if let Some(first_verifier) = allocation.signers.0.first() {
                 signing_address.clone_from(&first_verifier.signing_address);
-                message_cid.clone_from(&first_verifier.message_cid);
+                message_cid.clone_from(&first_verifier.message_cids.message_cids);
+                increase_allowance_cid = first_verifier.message_cids.increase_allowance_cid.clone();
             }
         }
 
+        let additional_status_message = increase_allowance_cid
+            .clone()
+            .map_or("".to_string(), |increase_allowance_cid| {
+                format!(", and here {}", increase_allowance_cid)
```

Comment:
```suggestion
                format!(", and here https://filfox.info/en/message/{}", increase_allowance_cid)
```
Or is it added somewhere else already and I missed it?